.gitignore

```diff
@@ -17,11 +17,14 @@ extras/automation/geckodriver.exe
 extras/automation/config.yaml
 src/sdkconfig.h
 sdkconfig.esp32dev
+docs/electroCad/fp-info-cache
+docs/electroCad/SandBox.sch-bak
+docs/electroCad/ser4.bck
+docs/electroCad/ser4.sch-bak
+docs/electroCad/ser4.kicad_pcb-bak
+docs/electroCad/MainComputer.sch-bak
+docs/electroCad/SteeringWheel.sch-bak
```

Comment:
Couldn't we ignore all files i.e. using `*` operator?

Reply:
removed it completely --> EasyEda-CAD